core/src/main/java/org/apache/iceberg/rest/requests/UpdateTableRequest.java

```diff
@@ -95,6 +110,12 @@ public Builder(TableMetadata base, boolean isReplace) {
       this.isReplace = isReplace;
     }
 
+    public Builder withIdentifier(TableIdentifier ident) {
```

Comment:
How about `forTable` or something that signals that this is a way to identify the table to change?

Reply:
`forTable` sounds good